src/profiles/profiles.db.ts

```diff
@@ -528,6 +524,16 @@ export class ProfilesDb extends LazyDbAccessCompatibleService {
     );
   }
 
+  async updatePrimaryAddress(param: {
+    profileId: string;
+    primaryAddress: string;
+  }) {
+    await this.db.execute(
+      `update ${PROFILES_TABLE} set primary_wallet = :primaryAddress where external_id = :profileId`,
```

Comment:
Use ALLCAPS for the SQL keywords, as per tradition?

Reply:
hm, seems we are mixed on this convention across the codebase. your thoughts, @prxt6529? Clean it up we go, or don't worry about it now?